services/app-web/src/pages/SubmissionSummary/SubmissionSummary.tsx

```diff
@@ -179,6 +180,11 @@ export const SubmissionSummary = (): React.ReactElement => {
                                     className={
                                         pkg.mccrsID ? styles.editLink : ''
                                     }
+                                    aria-label={
+                                        pkg.mccrsID
```

Comment:
nit: I try to use a shared variable when I'm trying to align display text with aria-label text so that these cannot accidentallynot diverge